runtime/redirector/generated.c.m4

```diff
@@ -332,8 +243,8 @@ JVM_LoadSystemLibrary(const char *libName)
  * com.sun.tools.attach.VirtualMachine support
  *
  * Initialize the agent properties with the properties maintained in the VM.
- * The following properties are apparently set by the reference implementation:
- *  sun.java.command = name of the main class
+ * The following properties are apparently set by the Sun implementation:
+ * 	sun.java.command = name of the main class
```

Comment:
Seems the spacing was accidentally? messed up here.

Reply:
Looks like bogus changes introduced by automated rebase algorithm